src/main/java/fi/hsl/transitdata/vehicleposition/application/gtfsrt/GtfsRtOccupancyStatusHelper.java

```diff
@@ -60,6 +60,8 @@ public Optional<GtfsRealtime.VehiclePosition.OccupancyStatus> getOccupancyStatus
             //Currently occu is only available for Suomenlinna ferries
             if (hfpPayload.getOccu() != 0) {
```

Comment:
Are negative values possible? Should this be 
```suggestion
            if (hfpPayload.getOccu() > 0) {
```

Reply:
I made the change you suggested and added a unit test as well.